src/controllers/director.residential.address.controller.ts

```diff
@@ -86,7 +85,12 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     } else if (selectedSraAddressChoice === "director_correspondence_address") {
       officerFilingBody.residentialAddress = officerFiling.serviceAddress;
       await patchOfficerFiling(session, transactionId, submissionId, officerFilingBody);
-      nextPageUrl = urlUtils.getUrlToPath(DIRECTOR_PROTECTED_DETAILS_PATH, req); //broken as agreed with BA
+      if(officerFiling.isMailingAddressSameAsRegisteredOfficeAddress){
```

Comment:
should we be checking `officereFiling.isMailingAddressSameAsHomeAddress` instead here? 
as we want to redirect if home address === correspondence address?

Reply:
This is to prevent two links being set. This is the home address link which should not be set if registered office address link is set, hence we skip the link page.

Reply:
nvm I think I got it, my apologies.